tests/Makefile

```diff
@@ -61,10 +61,10 @@ run_test: package venv_dev
 		nosetests $(NOSECFG) ${TEST} \
 		&& { ${SHUTDOWN_SERVER_CMD}; } || ${EXIT_ERROR}
 
-test_unit: venv_dev
+test_unit: package venv_dev
```

Comment:
Are these changes intentional here? Seems like a rebase issue with the ongoing changes to the testing infrastructure, #1414.

Reply:
Yes, because *source_code_comment_handler.py* depends on the generated thrift files and the package layout file: https://github.com/csordasmarton/codechecker/blob/e520ff027f6c4d46a91bfc01d25862e902f32f4a/libcodechecker/source_code_comment_handler.py

Reply:
I'll let @gyorb make a verdict on this as he is the one doign #1414. The rest looks good.

Reply:
I would prefer if not the whole package should be built for the unit tests. We could have another `thirft_test` target which could generate the files into a directory which is automatically detected and added to the PYTHONPATH by nosetest. In that case the package layout won't be needed.

With the package as a dependency it we be rebuilt for each unit test run.

Reply:
I removed the dependency from the source code comment handler, so it is no longer depend on the generated thrift files.